manifests/g/GeoGebra/GraphingCalculator/6.0.814/GeoGebra.GraphingCalculator.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Automatically updated by the winget bot at 2023/Nov/23
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: GeoGebra.GraphingCalculator
+PackageVersion: 6.0.814
+PackageLocale: en-US
+Publisher: International GeoGebra Institute
+PublisherUrl: https://www.geogebra.org
+# PublisherSupportUrl: https://help.geogebra.org/topic/help-support
+PrivacyUrl: https://www.geogebra.org/privacy
+Author: International GeoGebra Institute
+PackageName: GeoGebra Graphing Calculator
+PackageUrl: https://www.geogebra.org
+License: Non-commercial freeware; portions under GPL, CC-BY-NC-SA
+LicenseUrl: https://www.geogebra.org/license
+Copyright: © 2021 GeoGebra
+CopyrightUrl: https://www.geogebra.org/tos
+ShortDescription: Graph functions, investigate equations, and plot data with the free graphing app

```

```suggestion
ShortDescription: Graph functions, investigate equations, and plot data with the free graphing app.
```
Test change please ignore.